src/coreclr/jit/compiler.h

```diff
@@ -9553,7 +9553,8 @@ class Compiler
     //
     bool canUseEvexEncoding() const
```

Comment:
There's also `DoJitStressEvexEncoding()` which needs to also check for `Avx10v1`

Reply:
True. I have made this change.